frontend/src/views/SiteMap.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="text-left">
+    <h1 class="font-weight-black my-8">Plan du site</h1>
+    <v-row>
+      <v-col v-for="group in mapGroups" :key="group.title" cols="6">
+        <h2 class="my-2">{{ group.title }}</h2>
+        <ul>
+          <li v-for="link in group.links.filter((l) => !l.props)" :key="link.text">
+            <router-link :to="{ name: link.name }" :href="link.path">
+              {{ (link.meta || {}).title || link.name }}
+            </router-link>
+          </li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script>
+import { routes } from "@/router"
+export default {
+  name: "SiteMap",
+  data() {
+    let mcRoutes = JSON.parse(JSON.stringify(routes))
```

Comment:
Normalement on ne devrait pas dévoiler les routes réservées pour les utilisateurs identifiés au grand public (d'où le fait d'afficher un _404_ au lieu d'un _Vous devez être identifié.e pour accéder à cette page_). Donner l'information sur l'existance d'une route à quelqun.e qui n'est pas censé l'accéder peut-être un souci de sécurité.

Reply:
@hfroot J'ai pris la liberté d'ajouter un petit commit ici pour ne pas afficher les routes qui envoient vers un _not found_ : https://github.com/betagouv/ma-cantine/pull/1702/commits/1e2676e72c1dbb80a3ad2123514d73d8ecc2ee7c

Le souci était celui-ci :
![not-found-sitemap](https://user-images.githubusercontent.com/1225929/182401307-a33b1973-686f-4e49-b8f1-713f7fb4a967.gif)